### DISCUSSION TITLE
Regex link doesn't work

### DISCUSSION THREAD

**Discussion by SteveDunn** [community/community#9979] [Other Feature Feedback, Questions, & Ideas] [answered]
On the home page, there's examples. One of them is regex. One of those regex examples is a link: https://cs.github.com/?q=%2F%3CRoute(%08%7C%3E%2F

When following that link, I see:

![image](https://user-images.githubusercontent.com/263416/148949295-dbf00630-a324-416f-bdf0-ee8b60344b62.png)

**Comment by colinwm** [comment, marked as answer]
This is fixed now! Thanks for reporting.
